tools/toolchain/scripts/stage6/install_deepmd.sh

```diff
@@ -37,7 +37,11 @@ case "$with_deepmd" in
       if [ -f ${deepmd_pkg} ]; then
         echo "${deepmd_pkg} is found"
       else
-        download_pkg_from_cp2k_org "${deepmd_sha256}" "${deepmd_pkg}"
+        # download_pkg_from_cp2k_org "${deepmd_sha256}" "${deepmd_pkg}"
+        # TODO: saved to cp2k.org
+        wget ${DOWNLOADER_FLAGS} --quiet \
```

Comment:
I've put [a copy](https://www.cp2k.org/static/downloads/libdeepmd_c-2.2.9.tar.gz) on our server, please switch back to `download_pkg_from_cp2k_org`.